src/app/courses/courses-list/courses-list.component.html

```diff
@@ -1,10 +1,13 @@
 <ul class="coursesList">
-    <div class="noCourses" *ngIf="courses.length === 0">No data feel free to enter new course</div>
-    <li *ngFor="let item of filterItems(courses, searchValue) | orderBy; trackBy: trackByFn">
-        <app-course-item [id]="item.id" [title]="item.title" [description]="item.description" [duration]="item.duration"
-            [creationDate]="item.creationDate" [topRated]="item.topRated" (deletedItemId)="onDeleteCourse($event)">
-        </app-course-item>
-    </li>
+    <div class="noCourses" *ngIf="listOfCourses.length === 0">No data feel free to enter new course</div>
+    <div *ngIf="listOfCourses">
```

Comment:
You can try to use use _else_ here and move the _noCourses_ to _ng-template_ tag

Reply:
done